### PR TITLE
Entity Framework Support Policies

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

**Review by roji — APPROVED** [review]

File: entity-framework/efcore-and-ef6/support.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- The first version of Entity Framework and Entity Framework 4 are fully contained in the .NET Framework
+- Entity Framework 4.1, 4.2, 4.3, and 5.0 have some code in the .NET Framework, and some code shipped as NuGet packages
+- Entity Framework 6.0, 6.1, 6.2, 6.3, and 6.4 are shipped entirely as NuGet packages
+- Entity Framework Core (all versions) is an entirely separate codebase and ships as NuGet packages
+
+Support policies for each of these variations is described in this document. In all cases, the support policy applies to the latest patch of the given versions.
+
+## Entity Framework Core
+
+New versions of Entity Framework Core are shipped at the same time as new .NET versions. The Entity Framework Core support policy aligns with the [.NET support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core). See [current and planned releases](xref:core/what-is-new/index) for more information.
```
[the remark this answers is not in the document]
Comment (ajcvickers) [reply entity-framework/efcore-and-ef6/support.md:22]: Will do.

**Review by JeremyLikness — APPROVED** [review]
LGTM

**Comment by JeremyLikness** [comment]
@ajcvickers the CLA step was stuck so I forced a minor change to clear it.

**Review by Pilchie — APPROVED** [review]

**Pull request merged**